client/src/locales/en-US.json

```
@@ -1632,5 +1640,18 @@
                 "download": "Download"
             }
         }
+    },
+    "workspaceHistory": {
+        "actions": {
+            "restore": "Restore",
+            "cancel": "Cancel"
+        },
+        "restoreConfirmTitle": "Restore files",
+        "restoreConfirmQuestion": "This will restore one file or folder. It will overwrite your current files if present. Are you sure? | This will restore multiple files or folders. It will overwrite your current files if present. Are you sure?",
```

I would mention that the current versions can still be retrieved. Something like:
```suggestion
        "restoreConfirmQuestion": "Are you sure to restore the selected file or folder? This will overwrite any current version but you can still retrieve it in the History. | Are you sure to restore the selected files or folders? This will overwrite any current version but you can still retrieve them in the History.",
```